en/backup-to-aws-s3-using-br.md

```diff
@@ -404,6 +404,8 @@ Depending on which method you choose to grant permissions to the remote storage,
       schedule: "*/2 * * * *"
       backupTemplate:
         backupType: full
+        # Clean outdated backup data based on maxBackups or maxReservedTime, default policy is Retain
```

Comment:
Is there any special reason that you only add `cleanPolicy` to the AccessKey/SecretKey method?
If a user grant permission by IAM, do they need to configure `cleanPolicy`?

Plus, it might be more appropriate to just refer users to [Backup and Restore CR](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/backup-restore-cr#%E9%80%9A%E7%94%A8%E5%AD%97%E6%AE%B5%E4%BB%8B%E7%BB%8D:~:text=%E8%B0%83%E5%9B%9E%E5%8E%9F%E6%A0%B7%E3%80%82-,.spec.cleanPolicy,-%EF%BC%9A%E5%A4%87%E4%BB%BD%E9%9B%86%E7%BE%A4%E5%90%8E)